lldb/include/lldb/Interpreter/Interfaces/ScriptedPlatformInterface.h

```diff
@@ -19,11 +19,11 @@
 namespace lldb_private {
 class ScriptedPlatformInterface : virtual public ScriptedInterface {
 public:
-  StructuredData::GenericSP
+  virtual llvm::Expected<StructuredData::GenericSP>
   CreatePluginObject(llvm::StringRef class_name, ExecutionContext &exe_ctx,
                      StructuredData::DictionarySP args_sp,
-                     StructuredData::Generic *script_obj = nullptr) override {
-    return {};
+                     StructuredData::Generic *script_obj = nullptr) {
+    llvm_unreachable("unimplemented!");
```

Comment:
Could this be a pure virtual method? If not maybe something like "Cannot create an instance of an interface" would be a more informative error message.

Reply:
This can't be virtual pure because the `ScriptInterpreter` base class implements `virtual lldb::Scripted{Process,Platform,Thread}Interface()` methods that instantiate each base interface.